### ISSUE TITLE
Automatic quotes for attributes in Razor files

### ISSUE THREAD

**Issue by alex-jitbit** [dotnet/vscode-csharp#7418]
When editing plain HTML files, VSCode insert quotes automatically after typing tag attribute, like `<a class=` double quotes are inserted.

Please add the same when editing .CSHTML (Razor) files

**Comment by JoeRobich** [comment]
@davidwengier This looks like it is part of the language configuration already.
https://github.com/dotnet/vscode-csharp/blob/510d78ece2b1c6b1f37abeda6dcee84ca01fa77f/src/razor/language-configuration.json#L11-L18

**Comment by alex-jitbit** [comment]
@JoeRobich Nope, I'm not talking about auto-closing pairs. I'm talking about inserting quotes after typing `=` (equal sigh) in html attributes.
